core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -93,14 +93,10 @@ private[ui] class ExecutorsPage(
       </table>
 
     val content =
-      <div class="row-fluid">
+      <div class="row">
         <div class="span12">
-          <ul class="unstyled">
-            <li><strong>Memory:</strong>
-              {Utils.bytesToString(memUsed)} Used
```

Comment:
memUsed and maxMem locals in this function aren't being used anymore so lets remove them.